packages/live-share-canvas/src/core/LiveCanvas.ts

```diff
@@ -775,20 +772,26 @@ export class LiveCanvas extends DataObject {
 
     private updateCursorPosition(
         clientId: string,
-        userInfo?: IUserInfo,
+        eventUserInfo?: IEventUserInfo,
         position?: IPoint
     ) {
-        if (this._inkingManager) {
-            if (position) {
-                const liveCursor = this.getCursor(clientId, userInfo);
-
-                const screenPosition =
-                    this._inkingManager.viewportToScreen(position);
-
-                liveCursor.setPosition(screenPosition);
-            } else {
-                this.removeCursor(clientId);
-            }
+        if (position) {
+            LiveShareClient.getClientInfo(clientId).then((clientInfo) => {
+                if (this._inkingManager) {
+                    const userInfo: IUserInfo = {
+                        displayName: clientInfo?.displayName,
```

Comment:
I've learned from one key partner that they have intentionally not implemented `displayName` when implementing the previous callback, opting just to use profile picture. Spoofing is less of a concern for them though they are considering that. In any case, might be worth adding a flag/setting to hide display names.

Reply:
Let's take this up in a separate PR